src/components/Utils/UseLocalStorage/UseLocalStorage.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState, useEffect } from 'react';
+
+export function useLocalStorage(key: string, defaultValue: string) {
+  const [value, setValue] = useState(() => {
+    const stored = localStorage.getItem(key);
+    return stored !== null ? stored : defaultValue;
```

Comment:
```suggestion
    return stored ?? defaultValue;
```
we could use `nullish coalescing` there